kymatio/scattering1d/core/scattering1d.py

```diff
@@ -1,4 +1,5 @@
-def scattering1d(x, pad, unpad, backend, J, psi1, psi2, phi, pad_left=0,
+
+def scattering1d(x, pad, unpad, backend, J, log2_T, psi1, psi2, phi, pad_left=0,
```

Comment:
`J` is no longer used and can be removed.

Reply:
Agreed, but in this or separate PR?